app/lib/mini-course.cjsx

```diff
@@ -153,8 +157,19 @@ module.exports = React.createClass
 
       @props.projectPreferences.update "preferences.minicourses.completed_at.id_#{@props.minicourse.id}": now
       @props.projectPreferences.save()
+
+      @logToGeordi now
```

Comment:
The GS mini-course is fairly long. I'll add my test project to the list of projects setup to track to make sure this works.